src/modules/helm.rs

```diff
@@ -0,0 +1,119 @@
+use super::{Context, Module, RootModuleConfig};
+
+use crate::configs::helm::HelmConfig;
+use crate::formatter::StringFormatter;
+use crate::utils;
+
+/// Creates a module with the current Helm version
+///
+/// Will display the Helm version if any of the following criteria are met:
+///     - Current directory contains a `helmfile.yaml` file
+///     - Current directory contains a `Chart.yaml` file
+pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
+	let is_helm_project = context
+		.try_begin_scan()?
+		.set_files(&["helmfile.yaml", "Chart.yaml"])
+		.set_folders(&["templates"])
+		.is_match();
+
+	if !is_helm_project {
+		return None;
+	}
+
+	let mut module = context.new_module("helm");
+	let config = HelmConfig::try_load(module.config);
+	let parsed = StringFormatter::new(config.format).and_then(|formatter| {
+		formatter
+			.map_meta(|var, _| match var {
+				"symbol" => Some(config.symbol),
+				_ => None,
+			})
+			.map_style(|variable| match variable {
+				"style" => Some(Ok(config.style)),
+				_ => None,
+			})
+			.map(|variable| match variable {
+				"version" => format_helm_version(
+					&utils::exec_cmd("helm", &["version", "--short", "--client"])?
+						.stdout
+						.as_str(),
+				)
+					.map(Ok),
+				_ => None,
+			})
+			.parse(None)
+	});
+
+	module.set_segments(match parsed {
+		Ok(segments) => segments,
+		Err(error) => {
+			log::warn!("Error in module `helm`:\n{}", error);
+			return None;
+		}
+	});
+
+	Some(module)
+}
+
+fn format_helm_version(helm_stdout: &str) -> Option<String> {
+	// `helm version --short --client` output looks like this:
+	// v3.1.1+gafe7058
+
+	let version = helm_stdout
+		// split into ["v3.1.1","gafe7058"]
+		.splitn(2, '+')
+		// return "v3.1.1"
+		.next()?;
+
+	Some(version.to_string())
+}
```

Comment:
Can you update this function to also support helm 2, this should just be a case of removing the prefixed `Client: ` from the version string.
```
helm version --short --client
Client: v2.16.9+g8ad7037
```

Reply:
I just changed this parser logic, now should be support both Helm 1 and 2 version commands, i guess. :)